README.md

```diff
@@ -21,19 +21,54 @@ see https://stackoverflow.com/questions/71262775/how-do-i-ensure-assets-are-pres
   - postgres
   - yarn
 
-You can install postgres with `brew install postgresql` if you have homebrew. Then please run `bundle exec rails db:create` to set up the development and test databases.
+## Setting up the app
 
-Install dependencies:
+You can use [Homebrew](https://brew.sh/) to install any dependencies. The Homebrew documentation has lots of useful commands.
+
+If using Homebrew:
+
+Install postgres. You will need to select a version if using brew. Run the below command, changing the version number as appropriate.
+
+```
+brew install postgresql@14
+```
+
+You will be prompted on the command line to start the server with something like:
 
 ```
+brew services start postgresql@14
+```
+
+Install [Bundler](https://bundler.io/). You may run into an error where you need to update your Ruby version. Use rbenv or another version manager to do this.
+
+```
+gem install bundler
 bundle install
+```
+
+Create the development and test databases
+
+```
+bundle exec rails db:create
+```
+
+Install [Yarn](https://classic.yarnpkg.com/en/) (you can use Homebrew for this) and run the below:
+
+```
+brew install yarn
+yarn install
 yarn build
 yarn build:css
 ```
+
 ## Tests
 
 We test with RSpec and enforce 100% line and branch coverage with SimpleCov.
 
+You can run tests with the command `bundle exec rspec`. 
+
+In order to get tests running, you will need to make sure you have set up the correct environment config. Another developer can pair with you to make sure you have the correct config in your local .env and .env.test files.
```

Comment:
This isn't quite accurate. To get the tests running you shouldn't need anything in an `.env.test` file. But you do need _not_ to have anything in your `.env` file. This has tripped us up in the past, so I think it's worth spelling it out here: `.env` files are kinda dangerous, because they apply to both the development environment and the test environment. Better to copy the pre-populated `.env.sample` to `.env.development`, and you can then monkey with that in development to your heart's content, and then not have either a `.env` or ` .env.test` file.